### ISSUE TITLE
recv_message, caught exception

### ISSUE THREAD

**Issue by fcjxxl** [novnc/noVNC#688]
I use vnc.html connect localhost failed, below is the console log:

00:11:22.172 recv_message, caught exception: 
    name: URIError
    message: malformed URI sequence
Util.decodeUTF8@file:///D:/Program%20Files/noVNC-master/core/util.js:188:12
RFB.prototype._negotiate_server_init@file:///D:/Program%20Files/noVNC-master/core/rfb.js:969:29
RFB.prototype._init_msg@file:///D:/Program%20Files/noVNC-master/core/rfb.js:1081:28
RFB.prototype._handle_message@file:///D:/Program%20Files/noVNC-master/core/rfb.js:585:21
Websock.prototype._recv_message@file:///D:/Program%20Files/noVNC-master/core/websock.js:386:21
1util.js:44:47
Util.init_logging/Util.Error()util.js:44
Websock.prototype._recv_message()websock.js:413

so what's wrong?
ps: vncserver is ultravnc server, websockify is websockify.zip from https://github.com/kanaka/websockify/downloads, os is win10 x64

**Comment by DirectXMan12** [comment]
That's an ancient version of websockify (that link needs to be updated).  Please try with a newer version of websockify.

**Comment by fcjxxl** [comment]
At first I tried using python(2.7, 3.2, 3.5, I tried several versions) to run the latest version of websockify, but I met some problems like https://github.com/kanaka/websockify/issues/108(though there is a 08-no-sigchld-in-windows branch but still met some other problems, is there anyone can solve this?).
Then I followed the https://github.com/kanaka/websockify/wiki/Compiling-Websockify-as-Windows-Executable but still failed.
So finally I had to download websockify.exe from https://github.com/kanaka/websockify/downloads.
Currently I found only this way and websockify/other/js/websockify.js can run the websockify properly.
